--- a/node_modules/@expo/metro-config/build/ExpoMetroConfig.js
+++ b/node_modules/@expo/metro-config/build/ExpoMetroConfig.js
@@ -295,7 +295,7 @@
                 if (!platform) {
                     return [];
                 }
-                return require(path_1.default.join(getReactNativeHostPath(projectRoot, platform), 'rn-get-polyfills'))();
+                return require('@react-native/js-polyfills')();
             },
         },
         server: {
@@ -334,7 +334,7 @@
             asyncRequireModulePath: getExpoOptional(projectRoot, 'internal/async-require-module')
                 ? 'expo/internal/async-require-module'
                 : metroDefaultValues.transformer.asyncRequireModulePath,
-            assetRegistryPath: '@react-native/assets-registry/registry',
+            assetRegistryPath: 'react-native/asset-registry',
             // Determines the minimum version of `@babel/runtime`, so we default it to the project's installed version of `@babel/runtime`
             enableBabelRuntime: babelRuntimeVersion ?? undefined,
             // Allows additional babelrc lookups (mostly unused). The default of `undefined` enables the project's custom Babel config without enabling babelrc/configFile discovery
