--- a/node_modules/expo-modules-core/android/cmake/main.cmake
+++ b/node_modules/expo-modules-core/android/cmake/main.cmake
@@ -35,6 +35,7 @@
   ${REACT_NATIVE_INTERFACE_INCLUDE_DIRECTORIES}/react/fabric
   # header only imports from jni, e.g. react/turbomodule/CallInvokerHolder.h
   "${REACT_NATIVE_DIR}/ReactAndroid/src/main/jni"
+  "${REACT_NATIVE_DIR}/ReactCommon"
   "${ANDROID_SRC_DIR}/fabric"
   "${COMMON_DIR}"
   "${COMMON_DIR}/fabric"
