--- a/node_modules/expo-dev-launcher/ios/EXDevLauncherController.m
+++ b/node_modules/expo-dev-launcher/ios/EXDevLauncherController.m
@@ -3,7 +3,6 @@
 #import <React/RCTDevLoadingViewSetEnabled.h>
 #import <React/RCTDevMenu.h>
 #import <React/RCTDevSettings.h>
-#import <React/RCTRootContentView.h>
 #import <React/RCTAppearance.h>
 #import <React/RCTConstants.h>
 #import <React/RCTKeyCommands.h>
@@ -12,7 +11,6 @@
 #import <EXDevLauncher/EXDevLauncherManifestParser.h>
 #import <EXDevLauncher/EXDevLauncherRCTDevSettings.h>
 #import <EXDevLauncher/EXDevLauncherUpdatesHelper.h>
-#import <EXDevLauncher/RCTPackagerConnection+EXDevLauncherPackagerConnectionInterceptor.h>
 
 
 #import <ReactAppDependencyProvider/RCTAppDependencyProvider.h>
@@ -22,10 +20,6 @@
 #import <EXDevLauncher/EXDevLauncher-Swift.h>
 #else
 #import <EXDevLauncher-Swift.h>
-#endif
-
-#ifdef RCT_NEW_ARCH_ENABLED
-#import <React/RCTSurfaceView.h>
 #endif
 
 @import EXManifests;
@@ -188,6 +182,8 @@
 
 - (void)start:(id<EXDevLauncherControllerDelegate>)delegate launchOptions:(NSDictionary * _Nullable)launchOptions
 {
+  RCTBundleURLProviderAllowPackagerServerAccess(NO);
+
   _delegate = delegate;
   _launchOptions = launchOptions;
   NSDictionary *lastOpenedApp = [self.recentlyOpenedAppsRegistry mostRecentApp];
@@ -393,6 +389,8 @@
       onSuccess:(void (^ _Nullable)(void))onSuccess
         onError:(void (^ _Nullable)(NSError *error))onError
 {
+  RCTBundleURLProviderAllowPackagerServerAccess(YES);
+
   EXDevLauncherUrl *devLauncherUrl = [[EXDevLauncherUrl alloc] init:url];
   NSURL *expoUrl = devLauncherUrl.url;
   _possibleManifestURL = expoUrl;
@@ -540,11 +538,6 @@
     self.sourceUrl = bundleUrl;
 
 #if RCT_DEV
-    // Connect to the websocket, ignore downloaded update bundles
-    if (![bundleUrl.scheme isEqualToString:@"file"]) {
-      //[[RCTPackagerConnection sharedPackagerConnection] setSocketConnectionURL:bundleUrl];
-      RCTLogWarn(@"bundle scheme is file - unable to connect to sharedPackageConnection from EXDevLauncherController.");
-    }
     self.networkInterceptor = [[EXDevLauncherNetworkInterceptor alloc] initWithBundleUrl:bundleUrl];
 #endif
 
