--- a/node_modules/expo-modules-core/expo-module-gradle-plugin/src/main/kotlin/expo/modules/plugin/android/AndroidLibraryExtension.kt
+++ b/node_modules/expo-modules-core/expo-module-gradle-plugin/src/main/kotlin/expo/modules/plugin/android/AndroidLibraryExtension.kt
@@ -1,22 +1,21 @@
 package expo.modules.plugin.android
 
-import com.android.build.gradle.LibraryExtension
+import com.android.build.api.dsl.LibraryExtension
 
-internal fun LibraryExtension.applySDKVersions(compileSdk: Int, minSdk: Int, targetSdk: Int) {
+internal fun LibraryExtension.applySDKVersions(compileSdk: Int, minSdk: Int) {
   this.compileSdk = compileSdk
   defaultConfig {
     this@defaultConfig.minSdk = minSdk
-    this@defaultConfig.targetSdk = targetSdk
   }
 }
 
 internal fun LibraryExtension.applyLinterOptions() {
-  lintOptions.isAbortOnError = false
+  lint.abortOnError = false
 }
 
 internal fun LibraryExtension.applyPublishingVariant() {
-  publishing { publishing ->
-    publishing.singleVariant("release") {
+  publishing {
+    singleVariant("release") {
       withSourcesJar()
     }
   }
