--- a/node_modules/expo-modules-core/expo-module-gradle-plugin/src/main/kotlin/expo/modules/plugin/android/MavenPublicationExtension.kt
+++ b/node_modules/expo-modules-core/expo-module-gradle-plugin/src/main/kotlin/expo/modules/plugin/android/MavenPublicationExtension.kt
@@ -23,9 +23,7 @@
 import org.gradle.api.publish.PublicationContainer
 import org.gradle.api.publish.maven.MavenPublication
 import org.gradle.api.tasks.TaskProvider
-import java.nio.file.Path
-import kotlin.io.path.exists
-import kotlin.io.path.toPath
+import java.io.File
 
 internal data class PublicationInfo(
   val components: SoftwareComponent,
@@ -41,17 +39,15 @@
     artifactId = requireNotNull(project.androidLibraryExtension().namespace) {
       "'android.namespace' is not defined"
     },
-    version = requireNotNull(project.androidLibraryExtension().defaultConfig.versionName) {
-      "'android.defaultConfig.versionName' is not defined"
+    version = requireNotNull(project.version.toString().takeUnless { it == "unspecified" }) {
+      "'project.version' is not defined. Set `version = \"<x.y.z>\"` in the module's android/build.gradle."
     },
   )
 
-  fun resolvePath(repositoryPath: Path): Path {
+  fun resolvePath(repositoryPath: File): File {
     val groupPath = groupId.replace('.', '/')
     val artifactPath = "$groupPath/$artifactId/$version"
-    val publicationPath = repositoryPath.resolve(artifactPath)
-
-    return publicationPath
+    return File(repositoryPath, artifactPath)
   }
 
   override fun toString(): String {
@@ -180,7 +176,7 @@
 
   if (pathToRepository == null) {
     val mavenLocal = publishingExtension().repositories.mavenLocal()
-    val mavenLocalPath = mavenLocal.url.toPath()
+    val mavenLocalPath = File(mavenLocal.url)
     val publicationPath = publicationInfo.resolvePath(mavenLocalPath)
 
     if (!publicationPath.exists()) {
