--- a/node_modules/expo-dev-launcher/ios/Headers/RCTBundleURLProvider+Private.h
+++ b/node_modules/expo-dev-launcher/ios/Headers/RCTBundleURLProvider+Private.h
@@ -1,9 +1,2 @@
 // Copyright 2015-present 650 Industries. All rights reserved.
 
-#import <React/RCTBundleURLProvider.h>
-
-@interface RCTBundleURLProvider (EXDevLauncherRCTBundleURLProvider)
-
-- (NSString * _Nullable)guessPackagerHost;
-
-@end
