--- /dev/null
+++ b/node_modules/expo-modules-autolinking/android/expo-gradle-plugin/expo-autolinking-plugin/src/main/kotlin/expo/modules/plugin/GenerateInlineModulesTask.kt
@@ -0,0 +1,54 @@
+package expo.modules.plugin
+
+import org.gradle.api.DefaultTask
+import org.gradle.api.file.DirectoryProperty
+import org.gradle.api.provider.Property
+import org.gradle.api.tasks.Input
+import org.gradle.api.tasks.Internal
+import org.gradle.api.tasks.OutputDirectory
+import org.gradle.api.tasks.TaskAction
+import org.gradle.process.ExecOperations
+import javax.inject.Inject
+
+abstract class GenerateInlineModulesTask : DefaultTask() {
+  init {
+    group = "expo"
+    outputs.upToDateWhen { false }
+  }
+
+  @get:Internal
+  abstract val nodeWorkingDir: DirectoryProperty
+
+  @get:Internal
+  abstract val mirrorDirectory: DirectoryProperty
+
+  @get:Input
+  abstract val watchedDirectoriesSerialized: Property<String>
+
+  @get:OutputDirectory
+  abstract val outputDirectory: DirectoryProperty
+
+  @get:Inject
+  abstract val execOperations: ExecOperations
+
+  @TaskAction
+  fun generateInlineModules() {
+    execOperations.exec { spec ->
+      spec.workingDir(nodeWorkingDir.get().asFile)
+      spec.commandLine(
+        "node",
+        "--no-warnings",
+        "--eval",
+        "require('expo/bin/autolinking')",
+        "expo-modules-autolinking",
+        "mirror-kotlin-inline-modules",
+        "--kotlin-files-mirror-directory",
+        mirrorDirectory.get().asFile.absolutePath,
+        "--inline-modules-list-directory",
+        outputDirectory.get().asFile.absolutePath,
+        "--watched-directories-serialized",
+        watchedDirectoriesSerialized.get()
+      )
+    }
+  }
+}
