--- a/node_modules/expo-modules-autolinking/android/expo-gradle-plugin/expo-autolinking-plugin/src/main/kotlin/expo/modules/plugin/GeneratePackagesListTask.kt
+++ b/node_modules/expo-modules-autolinking/android/expo-gradle-plugin/expo-autolinking-plugin/src/main/kotlin/expo/modules/plugin/GeneratePackagesListTask.kt
@@ -2,11 +2,11 @@
 
 import expo.modules.plugin.configuration.ExpoModule
 import org.gradle.api.DefaultTask
-import org.gradle.api.file.RegularFileProperty
+import org.gradle.api.file.DirectoryProperty
 import org.gradle.api.provider.Property
 import org.gradle.api.tasks.Input
 import org.gradle.api.tasks.Internal
-import org.gradle.api.tasks.OutputFile
+import org.gradle.api.tasks.OutputDirectory
 import org.gradle.api.tasks.TaskAction
 
 /**
@@ -38,17 +38,17 @@
   lateinit var modules: List<ExpoModule>
 
   /**
-   * The output file where the package list should be written.
    */
-  @get:OutputFile
-  abstract val outputFile: RegularFileProperty
+  @get:OutputDirectory
+  abstract val outputDirectory: DirectoryProperty
 
   @TaskAction
   fun generatePackagesList() {
-    val target = outputFile.get().asFile
-    val content = generatePackageListFileContent()
-
-    target.writeText(content)
+    val target = outputDirectory.get().asFile
+      .resolve(namespace.get().replace('.', '/'))
+      .resolve(generatedPackageListFilename)
+    target.parentFile.mkdirs()
+    target.writeText(generatePackageListFileContent())
   }
 
   private fun generatePackageListFileContent(): String {
