--- a/node_modules/expo-dev-launcher/ios/EXDevLauncherController.h
+++ b/node_modules/expo-dev-launcher/ios/EXDevLauncherController.h
@@ -1,5 +1,4 @@
 #import <React/RCTBridgeModule.h>
-#import <React/RCTBridgeDelegate.h>
 
 #import <ExpoModulesCore/Platform.h>
 
@@ -39,7 +38,7 @@
 
 @end
 
-@interface EXDevLauncherController : RCTDefaultReactNativeFactoryDelegate <RCTBridgeDelegate, EXUpdatesExternalInterfaceDelegate>
+@interface EXDevLauncherController : RCTDefaultReactNativeFactoryDelegate <EXUpdatesExternalInterfaceDelegate>
 
 @property (nonatomic, weak) EXAppContext * _Nullable appContext;
 @property (nonatomic, strong) EXDevLauncherPendingDeepLinkRegistry *pendingDeepLinkRegistry;
